File: solved/hackerrank/Week of Code 24/Iterate It/gen.cc
#include <bits/stdc++.h>

using namespace std;

const int MVAL = 100;

int main(int argc, char **argv) {
  int n = (argc > 1) ? atoi(argv[1]) : 100;
  srand(time(0));
  printf("%d\n", n);
  for (int i = 0; i < n; ++i) {
    if (i) printf(" ");
    printf("%d", 1 + int(random() % MVAL));
  }
  puts("");
  return 0;
}

File: solved/Codechef/March Challenge 2018 DIV 1/Pishty and Triangles/gen.py
import sys
from random import randint

N, Q = int(sys.argv[1]), int(sys.argv[2])
print(N, Q)

MVAL = int(1e9)

for i in range(N):
    print(randint(0, MVAL), end=' ')

print()

for i in range(Q):
    if randint(0, 1) == 0:
        x, y = randint(0, N - 1), randint(0, N - 1)
        if (x > y):
            x, y = y, x
        print(2, x + 1, y + 1)
    else:
        pos, val = randint(0, N - 1), randint(0, MVAL)
        print(1, pos + 1, val)
